pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[tool.poetry]
+name = "climate-assessment"
+version = "0.1.4a0"
+description = "Climate assessment of long-term emissions pathways: IPCC AR6 WGIII version"
+authors = [
+    "Jarmo S. Kikstra <kikstra@iiasa.ac.at>",
+    "Zebedee Nicholls <zebedee.nicholls@climate-energy-college.org>",
+    "Jared Lewis <jared.lewis@climate-resource.com>",
+    "Christopher J. Smith <C.J.Smith1@leeds.ac.uk>",
+    "Robin D. Lamboll <r.lamboll@imperial.ac.uk>",
+    "Edward Byers <byers@iiasa.ac.at>",
+    "Marit Sandstad <marit.sandstad@cicero.oslo.no>",
+    "Laura Wienpahl <wienpahl@iiasa.ac.at>",
+    "Philip Hackstock <hackstock@iiasa.ac.at>",
+]
+readme = "README.rst"
+packages = [
+    {include = "climate_assessment", from = "src"},
+]
+keywords = ["climate", "integrated assessment", "emissions", "temperature", "IPCC"]
+license = "MIT License"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+]
+homepage = "https://github.com/iiasa/climate-assessment"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<3.12"
+aneris_iamc = "==0.3.1"
+dask = "==2024.4.1"
+fair = "==1.6.2"
+ixmp4 = "==0.8.0"
+joblib = "==1.3.2"
+openscm-runner = "==0.12.1"
+pandas = "==2.1.2"
+pooch = "==1.8.0"
+pyam-iamc = "==2.1.0"
+pymagicc = "==2.1.5"
+scmdata = "==0.17.0"
+silicone = "==1.3.0"
+toml = "^0.10.2"
```

Comment:
is there a reason for not explicitly pinning `toml`? Plus I think python 3.11 introduced `tomllib` as part of the standard library https://docs.python.org/3.11/library/tomllib.html, could be a good opportunity to kill off another dependency.

Reply:
Nope, just missed it. I'll try removing it.